packages/angular/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@documenso/embed-angular",
+  "version": "0.0.0",
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.cjs",
+  "module": "./dist/esm2022/index.mjs",
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/esm2022/index.mjs"
```

Comment:
👀

Reply:
The `esm2022` subdirectory, is it created by the build tooling or is that a choice of configuration on your end? If it's the latter it would be better to keep the flat structure, if it's the former then it doesn't matter.

Reply:
former